SPEC.md

```diff
@@ -0,0 +1,741 @@
+# Finality contract specification for the rollup integration
```

Comment:
Nit: Perhaps adding a reference to or comment about the original RFC in the `pm` repo is a good idea here.